MainModule/Server/Commands/Players.lua

```diff
@@ -935,14 +935,14 @@ return function(Vargs, env)
 				local adminDictionary, workspaceInfo = nil, nil
 				if elevated then
 					adminDictionary = {}
-					for _, v in pairs(service.GetPlayers()) do
+					for _, v in ipairs(service.GetPlayers()) do
```

Comment:
Isn't `ipairs()` less performance than `pairs()`? as `ipairs()` skips nil values?

service.GetPlayers will never return nil values in it's table. Using `ipairs()` has no benefit.

Reply:
`ipairs()` is more performant than `pairs()`

Reply:
> `ipairs()` is more performant than `pairs()`

that seems stupid but ok damn roblox and lua jank